mscore/palette.cpp

```diff
@@ -276,7 +282,7 @@ static void applyDrop(Score* score, ScoreView* viewer, Element* target, Element*
       DropData dropData;
       dropData.view       = viewer;
       dropData.pos        = pt;
-      dropData.dragOffset = pt;
+      dropData.dragOffset = QPointF();
```

Comment:
I'm not actually sure about this; maybe I should have left it alone.  It's only used for symbols apparently?